### PR TITLE
Adding checks

### PR DESCRIPTION
**Pull request by HernanW4**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    name: Test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        rust: [stable, beta, nightly]
+    steps:
+    - uses: actions/checkout@master
+    - name: Build & Test
+      run: |
+        sudo apt-get install -y libxcb-xfixes0-dev libxcb-shape0-dev
```
Comment (TroyNeubauer) [inline .github/workflows/main.yml]: We only needed these libs for flight tracking, so they can be deleted for this

**Review by TroyNeubauer — CHANGES_REQUESTED** [review]

**Comment by TroyNeubauer** [comment]
ping

**Review by TroyNeubauer — APPROVED** [review]
Looks good! We might want to get rid of the cargo run because in the future it will probably fail because of no input files, but that is a change for another day

**Comment by newcomb-luke** [comment]
I'd suggest, because we won't even be able to do cargo run until we can test the program by having it run on source files, we remove cargo run from the CargoBuild checks so that the checks can pass.
Alternatively we can just merge this and when we merge in the command_line branch it will be fixed

**Review by TroyNeubauer — APPROVED** [review]

**Pull request merged**